Signal/src/view controllers/MessagesViewController.m

```diff
@@ -1765,39 +1765,23 @@ - (void)audioRecorderDidFinishRecording:(AVAudioRecorder *)recorder successfully
 
 - (void)didPressAccessoryButton:(UIButton *)sender {
     [self dismissKeyBoard];
-
-    UIView *presenter = self.parentViewController.view;
-
-    [DJWActionSheet showInView:presenter
-                     withTitle:nil
-             cancelButtonTitle:NSLocalizedString(@"TXT_CANCEL_TITLE", @"")
-        destructiveButtonTitle:nil
-             otherButtonTitles:@[
-                 NSLocalizedString(@"TAKE_MEDIA_BUTTON", @""),
-                 NSLocalizedString(@"CHOOSE_MEDIA_BUTTON", @"")
-             ] //,@"Record audio"]
-                      tapBlock:^(DJWActionSheet *actionSheet, NSInteger tappedButtonIndex) {
-                        if (tappedButtonIndex == actionSheet.cancelButtonIndex) {
-                            DDLogVerbose(@"User Cancelled");
-                        } else if (tappedButtonIndex == actionSheet.destructiveButtonIndex) {
-                            DDLogVerbose(@"Destructive button tapped");
-                        } else {
-                            switch (tappedButtonIndex) {
-                                case 0:
-                                    [self takePictureOrVideo];
-                                    break;
-                                case 1:
-                                    [self chooseFromLibrary];
-                                    break;
-                                case 2:
-                                    [self recordAudio];
-                                    break;
-                                default:
-                                    break;
-                            }
-                        }
-                      }];
-}
+    
+    UIAlertController *alertController = [UIAlertController alertControllerWithTitle:NULL message:NULL preferredStyle:UIAlertControllerStyleAlert];
```

Comment:
We probably want an action sheet style alert controller.

And style nit: you should be passing in `nil` instead of `NULL` for things that can be an object. http://stackoverflow.com/questions/557582/null-vs-nil-in-objective-c